tests/openwisp2/urls.py

```diff
@@ -38,6 +38,8 @@
     url(r'^$', redirect_view, name='index'),
     url(r'^admin/', admin.site.urls),
     url(r'', include('openwisp_controller.urls')),
```

Comment:
I think we can start to get rid of `url` now and move to `path`, what do you think? :smile: